pyrr/geometric_tests.py

```diff
@@ -375,6 +387,7 @@ def sphere_does_intersect_sphere(s1, s2):
         return False
     return True
 
+
```

Comment:
There are still a lot of superfluous pep8 changes here.
Would be good if this was isolated to the change itself.
These can be done as part of #84